docs/udf.rst

```diff
@@ -402,6 +402,34 @@ An example code can be found `here <https://github.com/Open-EO/openeo-python-cli
 
 
 
+Logging from the UDFs
+==============================
+
+In some cases, users might want to use print statements within a UDF so that they can monitor their 
+results that makes debugging easier. One possible way is to use ``inspect`` logging function in the UDF.
+
+For example, in the previous example of rescaling an RGB image(from SENTINEL2_L2A), suppose user 
+want to keep a log of array shape encountered within a UDF::
+
+    # Create a UDF object from inline source code.
+    udf = openeo.UDF("""
+    from openeo.udf import XarrayDataCube
+    from openeo.udf.debug import inspect
+
+    def apply_datacube(cube: XarrayDataCube, context: dict) -> XarrayDataCube:
+        array = cube.get_array()
+        inspect(data=[array.shape], message="UDF logging shape of my array")
+        array.values = 0.0001 * array.values
+        return cube
+    """)
+
+Then, for users using a Jupyter notebook you can use the ```job.logs()``` to request the job logs where you will
```

Comment:
job.logs() will also work outside of jupyter notebooks, so can be generalized?

Reply:
Or maybe specify that the nice rendering only happens in jupyter, while in plain python you will get a dict.